.github/pull_request_template.md

```diff
@@ -1,3 +1,13 @@
 ### Summary:
 
 ### Test Plan:
+
+<!--
+  How did you validate that your changes were implemented correctly?
+-->
+
+- [ ] Wrote [automated tests](https://czi.atlassian.net/wiki/x/Hbl1H)
+- [ ] CI tests / new tests are not applicable
+- [ ] Manually tested my changes, but I want to keep the details secret
+- [ ] Manually tested my changes, and here are the details:
+  - Create an [alpha publish](https://github.com/chanzuckerberg/edu-design-system/blob/main/docs/PUBLISHING.md#alpha-release) and try out in `edu-stack` or `traject` as a sanity check if changes affect build or deploy, or are breaking.
```

Comment:
Is it mainly intuition for when this level of testing is needed or are there common things that have caused problems in the past and would be helpful to capture (if not already captured)?

Reply:
I think this is mostly useful for breaking changes, such as token changes, widely used component updates, hooks changes, major deps upgrades, which I can add to this bullet